packages/ui-breadcrumb/src/Breadcrumb/BreadcrumbLink/index.tsx

```diff
@@ -52,26 +56,43 @@ class BreadcrumbLink extends Component<BreadcrumbLinkProps> {
   handleRef = (el: Element | null) => {
     this.ref = el
   }
+  constructor(props: BreadcrumbLinkProps) {
+    super(props)
 
+    this.state = {
+      isTruncated: false
+    }
+  }
+  handleTruncation(isTruncated: boolean) {
+    if (isTruncated !== this.state.isTruncated) {
+      this.setState({ isTruncated })
+    }
+  }
   render() {
     const { children, href, renderIcon, iconPlacement, onClick, onMouseEnter } =
       this.props
-
+    const { isTruncated } = this.state
     const props = omitProps(this.props, BreadcrumbLink.allowedProps)
 
     return (
-      <Link
-        {...props}
-        href={href}
-        renderIcon={renderIcon}
-        iconPlacement={iconPlacement}
-        onClick={onClick}
-        onMouseEnter={onMouseEnter}
-        isWithinText={false}
-        elementRef={this.handleRef}
-      >
-        <TruncateText>{children}</TruncateText>
-      </Link>
+      <Tooltip renderTip={children} preventTooltip={!isTruncated}>
+        <Link
+          {...props}
+          href={href}
+          renderIcon={renderIcon}
+          iconPlacement={iconPlacement}
+          onClick={onClick}
+          onMouseEnter={onMouseEnter}
+          isWithinText={false}
+          elementRef={this.handleRef}
+        >
+          <TruncateText
+            onUpdate={(isTruncated) => this.handleTruncation(isTruncated)}
+          >
+            <span aria-hidden={isTruncated}>{children}</span>
```

Comment:
Saddly this is no good, NVDA ignores nodes that do not have text content or are set to `aria-hidden`. So here is just read 'button'